plugins/jobs/web_client/views/JobDetailsWidget.js

```diff
@@ -36,6 +36,13 @@ var JobDetailsWidget = View.extend({
             }
         });
 
+        this.listenTo(eventStream, 'g:eventStream.start', function (event) {
+            var status = this.job.get('status');
+            if (!JobStatus.finished(status)) {
+                this.job.fetch().done(() => { this.render(); });
```

Comment:
This is a personal nit, but I think these sorts of constructs are more readable when formatted as:
```
this.job.fetch()
    .done(() => { this.render(); });
```
If you disagree, then please disregard.